src/main/resources/com/puppycrawl/tools/checkstyle/checks/messages.properties

```diff
@@ -7,6 +7,7 @@ final.parameter=Parameter {0} should be final.
 forbid.escaped.unicode.char=Unicode escape(s) usage should be avoided.
 noNewlineAtEOF=File does not end with a newline.
 properties.duplicate.property=Duplicated property ''{0}'' ({1} occurrence(s)).
+properties.notSorted.property=Property key: ''{0}'' is not in right order with previous property ''{1}''.
```

Comment:
I don't think the `:` is needed. Violation should be a complete sentence and not 2 separate thoughts.
`not in right order with previous property` => `not in the right order with the previous property`

Reply:
I removed the : and corrected the missing "the"
The key name 0 and 1 i kept. I always search for it.
Done.